### PR TITLE
Hint toggle merge to dev

### PR DESCRIPTION
**Pull request by mukulikadey**
On the settings screen, there is now a hint toggle checkbox that allows the user to decide if they want the hint mode to be on or off. If the hint toggle is checked, then the hint will be displayed on the cards. If the hint toggle is unchecked, then the hint mode will be turned off.

### CODE REVIEW

**Comment by j-suguma** [comment]
@mukulikadey you're tests are failing:
image

its the error above in all theses tests:
image

**Comment by rekandiah** [comment]
Oh Muku I think the tests are failing because you need to change the version number.

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/SettingsScreen.java [hunk 1]
```diff
@@ -759,6 +769,24 @@ public void onClick(View v) {
                 df.setOnFileClickListener(aTypefaceEditFbListener);
                 df.show(getSupportFragmentManager(), "aTypefaceEditFB");
             }
+
+            if(v == hintToggleCheckbox){
+                SharedPreferences prefs = getApplicationContext().getSharedPreferences("AppPref", 0); // 0 for private mode
+                SharedPreferences.Editor editor = prefs.edit();
+                if(hintToggleCheckbox.isChecked()){
```
Comment (rekandiah) [inline app/src/main/java/org/liberty/android/fantastischmemo/ui/SettingsScreen.java]: fix spacing between your brackets

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/TwoFieldsCardFragment.java [hunk 2]
```diff
@@ -245,7 +246,21 @@ public int getCount() {
...
+
+    }
+
+    public void loadPrefs() {
+        SharedPreferences sp = getActivity().getSharedPreferences("AppPref", 0);
+        boolean hintToggleValue = sp.getBoolean("hintToggleCheck", true);
+
+        if(hintToggleValue == false){
+            field3CardPager.setVisibility(View.GONE);
+        }else{
```
Comment (rekandiah) [inline app/src/main/java/org/liberty/android/fantastischmemo/ui/TwoFieldsCardFragment.java]: spacing bro

**Comment by mukulikadey** [comment]
Spacing issues has been fixed :)

**Comment by j-suguma** [comment]
CI build passed:
image

tests:
image

**Comment by mukulikadey** [comment]
image

Hint Toggle Checkbox Espresso Test PASSED locally

**Pull request merged**